lib/mobile_application_platform/security_token/configuration.rb

```diff
@@ -78,6 +78,7 @@ def connection
         ) do |conn|
           conn.use :breakers
           conn.use Faraday::Response::RaiseError
+          conn.response :betamocks if Settings.mvi.mock
```

Comment:
instead of `Settings.mvi`, this will need to correspond to the value added in `settings.yml`, so will look something like `Settings.mobile_application_plaform.sts_token_mock`, or whatever you name it in that file

Reply:
Yep, good catch, I thought I changed it when I copied it over, but clearly missed it 😅